packages/module/src/Maintenance/Maintenance.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Button, EmptyState, EmptyStateBody, EmptyStateFooter, EmptyStateProps, EmptyStateStatus, EmptyStateVariant } from '@patternfly/react-core';
+import { HourglassHalfIcon } from '@patternfly/react-icons';
+
+/** extends EmptyStateProps */
+export interface MaintenanceProps extends Omit<EmptyStateProps, 'children' | 'title'> {
+    /** The title for the maintenance message */
+    titleText?: string;
+    /** Custom body text */
+    bodyText?: React.ReactNode;
+    /** A default bodyText used if no bodyText is provided. */
+    defaultBodyText?: React.ReactNode;
+    /** start time for maintenance UTC */
+    utcStartTime?: string;
+    /** end time for maintenance UTC */
+    utcEndTime?: string;
+    /** start time in a specific time zone */
+    startTime?: string;
+    /** end time in a specific time zone */
+    endTime?: string;
+    /** specify time zone */
+    timeZone?: string;
+    /** Information link */
+    redirectUrl?: string;
+    /** Information link title */
+    redirectUrlLinkText?: string;
+    /** Custom footer content */
+    customFooter?: React.ReactNode;
+    /** Custom OUIA ID */
+    ouiaId?: string | number;  }
+
+const Maintenance: React.FunctionComponent<MaintenanceProps> = ({ 
+  titleText = 'Maintenance in progress',
+  utcStartTime,
+  utcEndTime,
+  startTime,
+  endTime,
+  timeZone,
+  defaultBodyText = `We are currently undergoing scheduled maintenance and will be unavailable from ${utcStartTime} to ${utcEndTime} UTC (${startTime}-${endTime} ${timeZone}).`,
```

Comment:
What if consumer won't send the `startTime`, `endTime` and `endTime`? We would show to the customer `10am to 5pm UTC ()`. Can we make the times a bit automated? Meaning if consumer won't send it we won't show the partial?